js-api/src/widgets.ts

```diff
@@ -559,6 +560,27 @@ export class TabControl {
     api.grok_TabControlBase_Clear(this.dart);
   }
 
+  /** Add horisontal scroll to the header */
+  set headerScroll(scroll:boolean){
+    if (scroll)
+      this.header.className = 'd4-tab-header-stripe d4-tab-header-scroll';
+    else
+      this.header.className = 'd4-tab-header-stripe';
+  }
+
+  get headerScroll(): boolean{
+     return this.header.classList.contains('d4-tab-header-scroll');
+  }
+  /** Add navigation arrow to header */
+  set condensed(x:boolean){
+    if (x){
+      this.header.className = 'd4-tab-header-stripe d4-tab-header-condensed';
+    }
+    else{
+      this.header.className = 'd4-tab-header-condensed';
+    }
```

Comment:
rewrite it like
`this.header.className = x ? 'd4-tab-header-stripe d4-tab-header-condensed' : 'd4-tab-header-condensed';`
this would be shorter and clearer

Reply:
also spaces before braces